my_list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'MyEnumerable'
```

Comment:
- [ ] I couldn't see the expected result because you did not access `my_enumerable.rb` file properly. 👍 

```suggestion
require_relative 'my_enumerable'
```